main.go

```diff
@@ -17,10 +17,16 @@ limitations under the License.
 package main
 
 import (
+	"os"
+
 	"github.com/heptio/sonobuoy/cmd/sonobuoy/app"
 )
 
 // main entry point of the program
 func main() {
-	app.RootCmd.Execute()
+	err := app.RootCmd.Execute()
+	if err != nil {
+		// Execute takes care of printing the error
```

Comment:
Should we also log the error for consistency?

Reply:
We could but the user would see the same error twice unless we pass in some silencing flag to cobra.

The default output looks like this:

```
test salazar:sonobuoy cha$ go run main.go sdlkfjslkdjf
 Error: unknown command "sdlkfjslkdjf" for "sonobuoy"
Run 'sonobuoy --help' for usage.
ERRO[0000] unknown command "sdlkfjslkdjf" for "sonobuoy" 
exit status 1
```

I think it's cleaner without us logging the error as well as having cobra log the error.